Public/Src/Cache/ContentStore/Library/Service/LocalContentServerBase.cs

```diff
@@ -229,7 +229,7 @@ protected override async Task<BoolResult> StartupCoreAsync(OperationContext cont
 
             foreach (var store in StoresByName.Values)
             {
-                if (store is IContentStoreWithPostInitialization contentStore)
+                if (store is IContentStore contentStore)

```

Comment:
Isn't store guaranteed to be IContentStore?

Reply:
`TStore` is `IStartupShutdown` because the same base type is used to store both `IContentStore` and `IMemoizationStore` instances.